src/reflection.cpp

```diff
@@ -742,4 +742,13 @@ bool Verify(const reflection::Schema &schema, const reflection::Object &root,
   return VerifyObject(v, schema, root, flatbuffers::GetAnyRoot(buf), true);
 }
 
+bool VerifySizePrefixed(const reflection::Schema &schema,
+                        const reflection::Object &root, const uint8_t *buf,
+                        size_t length, uoffset_t max_depth /*= 64*/,
+                        uoffset_t max_tables /*= 1000000*/) {
+  Verifier v(buf, length, max_depth, max_tables);
+  return VerifyObject(v, schema, root, flatbuffers::GetAnySizePrefixedRoot(buf),
+                      true);
```

Comment:
nit: can you add a /*what_is_this_field=*/ comment to this boolean?

Reply:
should I change all that for Verify, which I just copied?